oauth2_client/providers/salesforce/provider.py

```diff
@@ -0,0 +1,98 @@
+"""
+SalesforceProvider.
+Currently supports only JWT Token Bearer OAuth2 flow.
+"""
+import json
+import requests
+
+from labsterutils.obj import get_value_or_default
+from oauth2_client.models import AccessToken
+from oauth2_client.providers.crypto import sign_data_rs256
+from oauth2_client.providers.oauth2.provider import OAuth2Provider
+
+
+class SalesforceProvider(OAuth2Provider):
```

Comment:
Having `IS_SANDBOX` doesn't help speed up things, as keys should be updated. 
URIs should be moved to Application.
As was found, JWT bearer grant type is not something SF specific, and after moving grant specific logic to a separate class (f.i., `JwtBearerGrant`), this provider will not make much sense.
Please remove Providers to keep things simple.
@labstersteve jfyi.

Reply:
providers removal in progress. Marking this as resolved.